persistence/src/main/java/de/cyface/persistence/DefaultPersistenceLayer.kt

```diff
@@ -531,23 +531,61 @@ class DefaultPersistenceLayer<B : PersistenceBehaviour?> : PersistenceLayer<B> {
                 }
             }
             return if (hasPressures && !forceGnssAscend) {
-                ascendFromPressures(tracks, PRESSURE_SLIDING_WINDOW_SIZE)
+                val altitudes = altitudesFromPressures(tracks, PRESSURE_SLIDING_WINDOW_SIZE)
+                totalAscend(altitudes)
```

Comment:
In this commit I had to make the intermediate state of the `ascendFrom**` method available, i.e. the 
`altitudesFrom**` to show the elevation profile in the client UI.